ground_station/src/hmi/hmi.cpp

```diff
@@ -317,12 +318,79 @@ void Hmi::testing() {
 
 /* DATA */
 
-void Hmi::initData() { window.initData(); }
+void Hmi::initData() {
+  dataFileCount = recorder.getFileCount();
+
+  // The display can show a maximum of 11 entries, no scrolling implemented!
+  // It is highly unlikely to get more than 11 flight logs because the flash is quite small
```

Comment:
if we do have more than 11 files, do we display only the newest or oldest files?

Reply:
They are sorted alphabetically, with how the flights are currently names, it would show the 11 oldest flights